### ISSUE TITLE
The full installation of pose inference was used, but after running `keypoints133.sh`, neither key points nor skeletons were drawn.

### ISSUE THREAD

**Issue by markryanwww** [facebookresearch/sapiens#98]
Hello, thank you very much for your excellent work. I have a question I'd like to ask you. After using `keypoints133.sh` for inference, why were the key points and skeleton not drawn for the hands and body, while only the face had the key points and skeleton drawn? My mmcv version is 2.0.0rc4, mmdet version is 3.2.0, mmpose version is 1.3.1, and torch version is 2.0.1.I’m eagerly awaiting your response.

**Comment by rawalkhirodkar** [comment]
@markryanwww You have not installed the modules provided with this repository. Please follow the full installation instructions in conda.sh to resolve this.

**Comment by markryanwww** [comment]
Hello, thank you very much for your response. I followed your instructions and reinstalled the provided modules, but I still couldn't complete the inference. I would like to know if the issue could be related to the Torch CUDA version, as I did not follow the conda.sh setup for that. It's also worth mentioning that I encountered the warning during the installation of pose: `warning: no files found matching 'mmpose/.mim/model-index.yml'`. Can I ignore this warning? Once again, thank you for your help.
